modules/stanford_layout_paragraphs/stanford_layout_paragraphs.module

```diff
@@ -95,3 +95,13 @@ function _stanford_layout_paragraphs_is_editing_layout_paragraphs() {
 function stanford_layout_paragraphs_paragraphs_behavior_info_alter(&$paragraphs_behavior) {
   $paragraphs_behavior['layout_paragraphs']['class'] = 'Drupal\stanford_layout_paragraphs\Plugin\paragraphs\Behavior\LayoutParagraphs';
 }
+
+/**
+ * Implements hook_form_layout_paragraphs_component_form_alter().
+ */
+function stanford_layout_paragraphs_form_layout_paragraphs_component_form_alter(&$form, &$form_state) {
+  // Modify the labels on the Spacer paragraph select list.
+  if (!empty($form['su_spacer_size'])) {
+    $form['su_spacer_size']['widget']['#options']['_none'] = 'Standard';
```

Comment:
this would be better suited in a widget form alter:
https://github.com/SU-SWS/stanford_profile_helper/blob/9.x/src/EventSubscriber/FormEventSubscriber.php#L48

Reply:
Ok by me.  I'll update it.